README.md

```diff
@@ -1,56 +1,59 @@
 killbill-client-php
 ===================
 
-PHP client library for killbill
+PHP client library for [Killbill](http://killbill.io)
 
 Configuration
 -------------
 
 In order to user the library, you need to:
 
-1. Download the library and unzip it in your application
```

Comment:
Could we keep some instructions for non-composer users? I can think of at least two deployments on legacy codebases that aren't using any package manager (code is checked-in and updated manually in the app tree).